bin/nodeCluster.bat

```diff
@@ -11,8 +11,8 @@ if "%ZLUX_NODE_LOG_DIR%" == "" (
   set ZLUX_NODE_LOG_DIR="../log"
 )
 call :makedir %ZLUX_NODE_LOG_DIR%
-set NODE_PATH=../../zlux-server-framework/js/node_modules;%NODE_PATH%
-cd ../js
+set NODE_PATH=../../zlux-server-framework/node_modules;%NODE_PATH%
```

Comment:
This is the most important line in all 4 of the repo PRs.
When you do an npm install for a "file:...", it ends up making a node_modules folder in your base folder, and doing a symlink. This is perfectly fine for development, but since this actually changes the structure, we need to test that the build system will do the right thing for packaging. pax files are supposed to preserve symlinks so perhaps its not an issue.
But, you could actually accomodate the case in which the symlinks didn't work by doing this instead:
`set NODE_PATH=../..;../../zlux-server-framework/node_modules;%NODE_PATH%` (../..; at front)

Reply:
OK, I get why you want to keep `../../zlux-server-framework/node_modules` but what exactly is the purpose of adding `../..`?